### PR TITLE
[CSFS-112] Develop a List View for Customer Orders And Store The Data in A Cookie

### PR DESCRIPTION
**Pull request by rubyade**
Screenshot (61)

### CODE REVIEW

File: submissions/rubyAdeniregun/php/orderpage.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+require_once("db_connection.php");
+
+echo "<h5> Order Page </h5>";
+
+setcookie("name", "json_encode($result)", time() + (86400 * 30), "/");
+
+try{
+    $sql= "SELECT * FROM order_details";
```
Comment (adaugochi) [inline submissions/rubyAdeniregun/php/orderpage.php:10]: The query you are to write for this task affects about 3 to 4 tables. That's why you were thought things like relationships on tables using foreign keys and the use of `join` statement

File: submissions/rubyAdeniregun/php/listview.php [hunk 2]
```diff
@@ -25,8 +25,8 @@
                 echo "<td>" . $row['email_address'] . "</td>";
                 echo "<td>" . $row['created_at'] . "</td>";
                 echo "<td>" . $row['actions'] . "</td>";
-                echo "<td><button type=button>View</button>" . $row['actions'] . "</td>";
-            echo "</tr>";
+                echo "<td><button type=button><a href='orderpage.php'>VIEW</a></button>" . $row['actions'] . "</td>";
```
Comment (adaugochi) [inline submissions/rubyAdeniregun/php/listview.php:28]: Also, the idea is to get all the order details of each customer you have on this page. Say for example you have Adaa as a customer, You are supposed to return details about customer Adaa when I click on the `View` button associated with Adaa

**Review by adaugochi — CHANGES_REQUESTED** [review]
For the last task, the issue I was pointing out wasn't fixed before the PR got `merged`. You make likely fix it along with this PR.

Well done @rubyade

File: submissions/rubyAdeniregun/php/orderpage2.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                // setting up the cookie to save the result gotten from the databse
+                    setcookie("customer_orders", json_encode($customer_orders), time() +3600);
+                    return $customer_orders;
+                } catch (PDOException $e) {
+                    die ('could not execute' . $sql . $e->getMessage());
+                }
+        }
+    }
+    //$data = new Data();
+    //$customer_orders = isset($_COOKIE['data']) ? unserialize($_COOKIE['data']) : $data->list($id);
```
Comment (Ireti203) [inline submissions/rubyAdeniregun/php/orderpage2.php:28]: Remove commented out codes

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**